osmaxx/core/templates/osmaxx/login_logout_link.html

```diff
@@ -1,13 +1,19 @@
 {% load i18n staticfiles %}
-
+<li role="presentation" class="col-lg-push-1 col-md-push-1">
 {% if user.is_authenticated %}
-<a href="{% url 'excerptexport:logout' %}?next={% url 'excerptexport:index' %}">
-    <img class="menu_icon" src='{% static "osmaxx/images/icon-logout.png" %}' alt="Logout" />
-    {% trans 'Logout' %} <strong>{{ user.username }}</strong>
-</a>
+    <a href="{% url 'excerptexport:logout' %}?next={% url 'excerptexport:index' %}">
+        <img class="menu_icon" src='{% static "osmaxx/images/icon-logout.png" %}' alt="Logout" />
+        {% trans 'Logout' %}
+    </a>
+</li>
+<li role="presentation" class="col-lg-push-1 col-md-push-1">
+    <a href="{% url 'profile:edit_view' %}">
+        <strong>{{ user.username }}</strong>
+    </a>
 {% else %}
 <a href="{% url 'excerptexport:login' %}?next={% url 'excerptexport:index' %}">
     <img class="menu_icon" src='{% static "osmaxx/images/icon-login.png" %}' alt="Login" />
     {% trans 'Login' %}
 </a>
 {% endif %}
+</li>
```

Comment:
Less redundancy (but alas, not really less template code)
```html
{% load i18n staticfiles capture_as %}
{% if user.is_authenticated %}
    {% capture_as direction %}out{% end_capture_as %}
{% else %}
    {% capture_as direction %}in{% end_capture_as %}
{% endif %}
<li role="presentation" class="col-lg-push-1 col-md-push-1">
    <a href="{% url 'excerptexport:log{{ direction }}' %}?next={% url 'excerptexport:index' %}">
        <img class="menu_icon" src='{% static "osmaxx/images/icon-log{{ direction }}.png" %}' alt="Log{{ direction }}" />
        {% trans 'Log{{ direction }}' %}
    </a>
</li>
{% if user.is_authenticated %}
<li role="presentation" class="col-lg-push-1 col-md-push-1">
    <a href="{% url 'profile:edit_view' %}">
        <strong>{{ user.username }}</strong>
    </a>
</li>
{% endif %}
```
(not a serious suggestion :stuck_out_tongue_winking_eye:)